page-optimize.php

```diff
@@ -4,13 +4,16 @@
 Plugin URI: https://wordpress.org/plugins/page-optimize/
 Description: Optimizes JS and CSS for faster page load and render in the browser.
 Author: Automattic
-Version: 0.1.3
+Version: 0.2.0
```

Comment:
Major bump, cache was a big thing.

I'd say next major would be the `WP_CONTENT_DIR` stuff.

Reply:
sounds good :)